Lib/fontbakery/specifications/googlefonts.py

```diff
@@ -1291,13 +1291,11 @@ def com_google_fonts_check_037(font):
     raise error
 
   xml_report = open("{}.report.xml".format(font), "r").read()
-  try:
-    os.remove("{}.report.xml".format(font))
-    os.remove("{}.report.html".format(font))
-  except:
-    # Treat failure to delete reports
-    # as non-critical. Silently move on.
-    pass
+
+  os.remove("{}.report.xml".format(font))
+  os.remove("{}.report.html".format(font))
+  fval_file = os.path.join(os.path.dirname(font), 'fval.xsl')
```

Comment:
Shouldnt we use the same os.path.join style for all 3 files? (or for none of them in case that's not really necessary ?)

@m4rc1e could you amend that, please?

Reply:
Agree, I'll modify the other two.